codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -209,7 +209,7 @@ class ServerProtocolTestGenerator(
             rust("""*http_request.uri_mut() = "${httpRequestTestCase.uri}?$queryParams".parse().unwrap();""")
         }
         httpRequestTestCase.host.orNull()?.also {
-            rust("""todo!("endpoint trait not supported yet");""")
+            rust("""/* todo!("endpoint trait not supported yet"); */""")
```

Comment:
I am not sure yet what is needed here so any directions would be useful

Reply:
Since we are at an early phase of development, we do not implement all Smithy protocol traits yet and we want this to hard fail because if a model requires functionalities that are not implemented, it should fail when it run.

Reply:
Ok so this is also not failing now that I added s3 to the block-list as suggested.

Reply:
So I reverted my change here.